src/components/Button/Button.tsx

```diff
@@ -0,0 +1,12 @@
+import {
+  Button as MuiButton,
+  ButtonProps as MuiButtonProps,
+} from "@mui/material";
+
+interface ButtonProps extends MuiButtonProps {}
```

Comment:
prefer not to use interfaces. Syntax will grow complex as we go further.

Instead, type syntax is much similar to our regular js
type ButtonProps = MuiButtonProps;

Reply:
done changed to type.